benchmarks/python/test_rope.py

```diff
@@ -176,3 +62,76 @@ def test_rope_benchmark(
 
     if not disable_benchmarking:
         run_benchmark(benchmark, fd.execute, inputs)
+
+
+@pytest.mark.parametrize(
+    "rope_variation",
+    [
+        "llama_2_7b_hf_rope",
+        "llama_3_8B_rope",
+        "hf_qwen2_rope",
+        "hf_phi3_rope",
+        "hf_mistral_nemo_rope",
+    ],
+)
+@pytest.mark.parametrize("executor", ["eager", "torchcompile", "thunder", "thunder-torchcompile"])
+def test_rope_variations_fwd_benchmark(
+    benchmark,
+    rope_variation: str,
+    executor: str,
+):
+    kwargs = {}
+    if executor == "thunder":
+        kwargs["nv_enable_matmul"] = True
```

Comment:
BTW, this is where I'm enabling matmul in nvfuser.

This gives us a single fusion region, I believe is something we would like. cc'ing @naoyam